test/multiverse/suites/view_component/view_component_instrumentation_test.rb

```diff
@@ -60,12 +60,12 @@ def test_error_raised
 
   # Test metric name being built when the controller class doesn't respond to :identifier
   # https://github.com/newrelic/newrelic-ruby-agent/pull/2870
-  def test_the_metric_name_omits_the_identifier_when_absent
+  def test_the_metric_name_records_default_name_on_error
```

Comment:
Should the comment on line 61 be updated?

Reply:
Yes! Great catch

Reply:
Added https://github.com/newrelic/newrelic-ruby-agent/pull/2907/commits/b7f97152908d993867a9ee2000853893deb090f8